src/System-BasicCommandLineHandler.package/BasicCommandLineHandler.class/instance/activate.st

```diff
@@ -1,8 +1,10 @@
 activation
 activate
 	
-	[ self handleArgument: (self arguments 
+	[ [ self handleArgument: (self arguments 
 			ifEmpty: [ '' ] 
 			ifNotEmpty: [ :arguments| arguments first ])] 
 		on: Exit 
-		do: [ :exit | ^ self handleExit: exit ]
\ No newline at end of file
+		do: [ :exit | ^ self handleExit: exit ] ]
+			forkAt: Processor userSchedulingPriority
```

Comment:
Does forking is really needed? Because #activate is called from class side #startUp method inside deferred action